chains/evm/voter/voter.go

```diff
@@ -28,42 +44,147 @@ type MessageHandler interface {
 }
 
 type EVMVoter struct {
-	mh             MessageHandler
-	client         ChainClient
-	fabric         calls.TxFabric
-	gasPriceClient calls.GasPricer
+	mh                   MessageHandler
+	client               ChainClient
+	fabric               calls.TxFabric
+	gasPriceClient       calls.GasPricer
+	pendingProposalVotes map[common.Hash]uint8
 }
 
-func NewVoter(mh MessageHandler, client ChainClient, fabric calls.TxFabric, gasPriceClient calls.GasPricer) *EVMVoter {
-	return &EVMVoter{
-		mh:             mh,
-		client:         client,
-		fabric:         fabric,
-		gasPriceClient: gasPriceClient,
+func NewVoter(mh MessageHandler, client ChainClient, fabric calls.TxFabric, gasPriceClient calls.GasPricer) (*EVMVoter, error) {
+	voter := &EVMVoter{
+		mh:                   mh,
+		client:               client,
+		fabric:               fabric,
+		gasPriceClient:       gasPriceClient,
+		pendingProposalVotes: make(map[common.Hash]uint8),
+	}
+
+	ch := make(chan common.Hash)
+	_, err := client.SubscribePendingTransactions(context.TODO(), ch)
```

Comment:
If node is not geth then this will fail. 
There should be a way that this done in more modular manner. For example adding boll field to config like gethNode = true/false (this is the easiest  way i guess)